.github/renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "config:best-practices",
+    ":gitSignOff",
+    ":rebaseStalePrs",
+    "group:allNonMajor",
+    "group:linters",
+    "group:test",
+    ":preserveSemverRanges",
+    ":pinOnlyDevDependencies"
+  ],
+  "labels": ["kind/dependency upgrade"],
+  "npm": {
+    "minimumReleaseAge": "1 day"
+  },
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "groupName": "GitHub Actions"
+    },
+    {
+      "matchPackagePatterns": ["^@backstage/"],
+      "groupName": "Core Backstage packages"
+    },
+    {
+      "matchPackagePatterns": [
+        "^@semantic-release/",
+        "^@semrel-extra/",
+        "^multi-semantic-release$"
+      ],
+      "groupName": "semantic-release monorepo"
```

Comment:
Same here  https://docs.renovatebot.com/presets-monorepo/#monoreposemantic-release

Reply:
so are you suggesting I remove these groupings?

Reply:
@nickboldt Removing these then right? Happy to ship after that